docker/checkout.bash

```diff
@@ -54,6 +54,8 @@ fi
 # NOTE: The ibeo_msgs package is ignored because on build the cmake files in that package run a sed command 
 #       which can make them incompatible with a new ros version after a source switch
 git clone https://github.com/astuff/astuff_sensor_msgs 
+#rosbridge_suite is a ROS meta-package including all the rosbridge packages.
+git clone https://github.com/RobotWebTools/rosbridge_suite.git
```

Comment:
This git clone is overwriting the astuff_sensor_msgs checkout. It needs to be placed either before or after that